e2e-tests/tests/init-deploy/01-create-cluster.yaml

```diff
@@ -1,11 +1,31 @@
-apiVersion: kuttl.dev/v1beta1
-kind: TestStep
-timeout: 300
-commands:
-- script: |
-    set -o errexit
-    set -o xtrace
+apiVersion: ps.percona.com/v2
+kind: PerconaServerForMySQL
+metadata:
+  name: init-deploy
+spec:
+  secretsName: test-secrets
+  sslSecretName: test-ssl
+  mysql:
+    image: percona/percona-server:8.0
+    imagePullPolicy: Always
 
-    source ../../functions
+    size: 3
+    sizeSemiSync: 0
 
-    get_cr | kubectl apply -n "${NAMESPACE}" -f -
```

Comment:
idea behind `get_cr` function - have control which MySQL, Orchestrator image is run during the release testing

Reply:
I see your point. I separated the test commit from this PR. Let's merge this and I'll revert `get_cr`